### PR TITLE
Pass the root path instead of base and cron dirs

### PR DESCRIPTION
**Pull request by albertodonato**
Trivial change to pass the root dir to get_path (used only in tests), instead of multiple paths

### CODE REVIEW

File: lib/charms/archive_auth_mirror/utils.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 from charmhelpers.core import host
 
 
-def get_paths(base_dir=None):
+def get_paths(root_dir=Path('/')):
```
Comment (bjornt) [inline lib/charms/archive_auth_mirror/utils.py]: Maybe defaulting to None would be better? Path('/') is probably immutable, but it feels safer to use None.

File: lib/charms/archive_auth_mirror/utils.py [hunk 2]
```diff
@@ -52,13 +51,9 @@ def get_virtualhost_config(hookenv=hookenv):
         'basic_auth_file': str(paths['basic-auth'])}
 
 
-def install_resources(base_dir=None, cron_dir=None):
+def install_resources(root_dir=Path('/')):
```
Comment (bjornt) [inline lib/charms/archive_auth_mirror/utils.py]: Same as above.

File: lib/charms/archive_auth_mirror/utils.py (hunk 1 above)
Comment (albertodonato) [inline lib/charms/archive_auth_mirror/utils.py]: Done

File: lib/charms/archive_auth_mirror/utils.py (hunk 2 above)
Comment (albertodonato) [inline lib/charms/archive_auth_mirror/utils.py]: Done

**Pull request merged**